src/Pages/Home/Home.jsx

```diff
@@ -0,0 +1,213 @@
+                the cultivation and perfection of human beings.
+              </span>
+
+              <button className="cursor bg-black-9 rounded-xs border-none  outline-none text-color-0 pd-x-4 pd-y-3 text-s ">
+                Explore Video
+              </button>
+            </div>
+          </div>
+        </section>
+        <section className=" flex  flex-column  pd-y-8 wt-80 m-auto justify-btw">
+          <div className="text-m text-semibold flex items-start pd-y-5">
+            <span>Recommend Video</span>
+          </div>
+          <div className="flex card-gap wrap">
+            <div className="box-shadow-1 wt-30 flex flex-column ">
+              <div className="wt-100 ">
+                <img
+                  src="./assets/farming.jpg"
+                  className="object-content wt-100 h-80 "
+                  alt=""
+                />
+              </div>
+
+              <div className="pd-x-4 flex flex-column">
+                <div className=" flex ">
+                  <span className="text-s flex items-start">
+                    Palm Oil Farming | सरकार उठाएगी खर्च | पाम तेल की खेती |
+                    Palm oil tree | Palm farming subsidy
+                  </span>
+                </div>
+                <div className=" flex items-start   text-semibold">
+                  <div className="flex items-center pd-y-3 pd-x-3">
+                    <img
+                      src="./assets/video.jpg"
+                      className="avatar rounded-full sm"
+                      alt="farming indian"
+                    />
+                    <span className="pd-x-2">Indian Farmer</span>
+                  </div>
+                </div>
+                <div className="flex justify-btw pd-3">
+                  <span>1 day ago</span>
+                  <div className="flex items-center ">
+                    <FaRegEye> </FaRegEye>
+                    <span>16.7k</span>
+                  </div>
+                </div>
+
+                <div className="flex flex-column pd-y-3 justify-btw ">
+                  <button className="cursor bg-black-9 rounded-xs border-none  outline-none text-color-0 pd-x-4 pd-y-3 text-s ">
+                    Watch Now
+                  </button>
+                </div>
+              </div>
+            </div>
+            <div className="box-shadow-1 wt-30 flex flex-column ">
+              <div className="wt-100 ">
+                <img
+                  src="./assets/farming.jpg"
+                  className="object-content wt-100 h-80 "
+                  alt=""
+                />
+              </div>
+
+              <div className="pd-x-4 flex  flex-column">
+                <div className=" flex ">
+                  <span className="text-s flex items-start">
+                    Palm Oil Farming | सरकार उठाएगी खर्च | पाम तेल की खेती |
+                    Palm oil tree | Palm farming subsidy
+                  </span>
```

Comment:
This info you should get from database. If you are going to do that next. I'll suggest do it first then make all things.

Reply:
yes after thus  will give to data base.